ambuda-org/ambuda
# Organize imports in python files with isort

Clean up of imports in python files. A diff to address https://github.com/ambuda-org/ambuda/issues/296

Test plan
1. Ran py-lint 
```
$ make py-lint
Running Python Black to check formatting
All done! ✨ 🍰 ✨
129 files left unchanged.
Running Python isort to organize module imports
Running Python flake8 to conform with PEP8
Python lint completed
```
2. make install
3. make devserver
4. Tested tabs (Texts, Dictionaries, About, Login, help).
Not tested:
1. Proofing.
